### DISCUSSION TITLE
Add a `$__teams` global variable

### DISCUSSION THREAD

**Discussion by ethirolle** [grafana/grafana#38196] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Add a global variable (`$__teams`) that contains a list of the teams that the current user is a member of.

**Why is this needed**:
This would be similar to the `$__org` global variable: https://grafana.com/docs/grafana/latest/variables/variable-types/global-variables/#__org. It would allow the Grafana admins to customize a dashboard to return different results (or use different datasources) based on team membership.

In the customer's words:

> Scenario: If we have per-team datasources and want to have a shared dashboard where the datasource points to that team's datasource automatically.
> The more generic way might be to be able to define custom template variables for a team
> So the template variable could be used as the datasource name or part of the mane

An alternative fix for this same problem is described in this issue, which asks for per-team variables: https://github.com/grafana/grafana/issues/21071

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by lbeaucourt** [comment]
I'm reopening this discussion, is there a way to access a member's teams using a global variable?

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82940. Feel free to continue the discussion around this there. Thank you!
